src/components/expandableSection/index.tsx

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash';
+import React, {useState} from 'react';
+import {LayoutAnimation} from 'react-native';
+import View from '../view';
+//@ts-ignore
+import ListItem from '../listItem';
+import Button from '../button';
+
+
+export type ExpandableSectionProps = {
+  /**
+   * expandableSection header element
+   */
+  sectionHeader?: JSX.Element;
+  /**
+   * expandableSection expandable children
+   */
+  children?: React.ReactNode;
+  /**
+   * expandableSection icon color
+   */
+  iconColor?: string;
```

Comment:
Don't support having a separate icon. 
Let the user pass the whole expand section element.

The user can decide how it looks and if they want an icon or not. 
See following comment, I explain better how it should integrate with private uilib.